### PR TITLE
DATA-37 - Report format changes.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR changes some things to the reports such as:

- Get only users with the role student for the three reports.
- Remove user role info.
- Add section name to last page accessed report.

## Reviewers:

- [ ] @andrey-canon

### CODE REVIEW

File: openedx_proversity_reports/reports/last_page_accessed.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 from openedx_proversity_reports.edxapp_wrapper.get_course_cohort import get_course_cohort
 from openedx_proversity_reports.edxapp_wrapper.get_course_teams import get_course_teams
 from openedx_proversity_reports.edxapp_wrapper.get_modulestore import get_modulestore
-from openedx_proversity_reports.utils import get_staff_user, get_user_role
```
Comment (andrey-canon) [inline openedx_proversity_reports/reports/last_page_accessed.py]: Should we remove` get_staff_user, get_user_role` from utils ?

Comment (Squirrel18) [reply openedx_proversity_reports/reports/last_page_accessed.py]: No.

Comment (andrey-canon) [reply openedx_proversity_reports/reports/last_page_accessed.py]: Which is the reason ?

Comment (Squirrel18) [reply openedx_proversity_reports/reports/last_page_accessed.py]: Proversity mindset.

**Pull request merged**